R/measures.R

```diff
@@ -47,16 +49,26 @@
 #'
 #' @examples
 #' \dontrun{
-#' riem_measures(station = "VOHY", date_start = "2000-01-01", date_end = "2016-04-22")
+#' riem_measures(station = "VOHY", date_start = "2000-01-01", date_end = "2016-04-22", trace = 1e-4, missing = NA)
 #' }
 riem_measures <- function(station = "VOHY",
                           date_start = "2014-01-01",
-                          date_end = as.character(Sys.Date())){
+                          date_end = as.character(Sys.Date()),
+                          trace = .0001,
+                          missing = NA){
 
-  base_link <- "https://mesonet.agron.iastate.edu/cgi-bin/request/asos.py/"
 
+  base_link <- "https://mesonet.agron.iastate.edu/cgi-bin/request/asos.py/"
 
+  if(!is.na(trace) & !(trace %in% list("T", "", .0001))) {
+    stop(call. = FALSE,
+         "trace must be one of NA, 'T', '' or 0.0001.")  # nolint
+  }
 
+  if(!is.na(missing) & !(missing %in% list("M", ""))) {
+    stop(call. = FALSE,
+         "missing must be one of NA, 'M', ''")  # nolint
```

Comment:
I don't understand, I had read "M, null, or empty"?

Reply:
Oh I guess that's what happens to the JSON? I suppose a test would help me see what's expected.

Reply:
OK, so - turns out I misread the original issue #22 , but I caught up! It now correctly passes those arguments (in their intended form) along to the GET request, and then handles the output. I've also added tests, and updated the DESCRIPTION and the NEWS file. Happy to add more tests if you think we should cover it more exhaustively.